dev/docs/analytics-header.html

```diff
@@ -0,0 +1,7 @@
+<!-- Google Tag Manager -->
```

Comment:
@jayoung-lee can you verify this looks right

Reply:
If this comes after the <head> tag, LTGM! :)

Reply:
It's been a while since I did web development, so I'm fuzzy on how GTM works. But I thought this script was creating the tag?

I can say that this will get inserted at the same place in the HTML document that the previous script was (the deleted dev/docs/analytics.html file).

Reply:
I'ts late Friday afternoon, this won't be merged until next week. I would be happy to chat then if you like.

Reply:
we sync'd offline, we're going to deploy and verify if it's working on master.